WebRoot/js/ajax/config/msgs/AjxMsg_vi.properties

```diff
@@ -0,0 +1,157 @@
+# 
```

Comment:
I removed the vi file since no other localized file was there. Only en-US.
I tried to mimic the en-US file structure and copied vi as well.